server/protocols/webadmin/webadmin-core/src/main/java/org/apache/james/webadmin/routes/HealthCheckRoutes.java

```diff
@@ -87,19 +87,15 @@ public Object validateHealthchecks(Request request, Response response) {
 		response.status(getCorrespondingStatusCode(anyUnhealthyOrDegraded));
 		return response;
 	}
-    
+
     @GET
     @Path(CHECKS)
     @ApiOperation(value = "List all health checks")
-    @ApiResponses(value = {
-            @ApiResponse(code = HttpStatus.OK_200, message = "List of all health checks", 
-            		response = HealthCheckDto.class, responseContainer = "List"),
-            @ApiResponse(code = HttpStatus.NOT_FOUND_404, message = "No health checks found")
-    })
-    public Object getHealthChecks(Request request, Response response) {
-		List<HealthCheckDto> checks = healthChecks.stream()
-				.map(healthCheck -> new HealthCheckDto(healthCheck.componentName())).collect(Guavate.toImmutableList());
-		return checks;
+    @ApiResponse(code = HttpStatus.OK_200, message = "List of all health checks",
+                response = HealthCheckDto.class, responseContainer = "List")
+    public Object getHealthchecks(Request request, Response response) {
+        return healthChecks.stream()
+            .map(healthCheck -> new HealthCheckDto(healthCheck.componentName())).collect(Guavate.toImmutableList());
```

Comment:
Can you add a line break before `.collect(...)` ? This would ease reading.

(for the record I thought you were returning directly the stream and that spark was handling it... :-) )